kw6/reader.py

```diff
@@ -46,21 +47,26 @@ def from_path(path: Path, header_path: Path = None):
         if version != "KW6FileClassVer1.0":
             raise ValueError(f"Unexpected file version {version}")
 
-        initial_position_header = Position.skip_(stream)
+        initial_position_header = PositionHeader.from_stream_(stream)
+
+        cached_byte_positions = (
+            hdr.positions(header_path)
```

Comment:
I realize now that this was not part of this PR but isnt `header` a more descriptive name than `hdr`? That's what we call it in kmm

Reply:
Sure

Reply:
But we also have two other "headers" already so I'm not sure it's great idea actually